src/parsers/manifest/smooth/representationIndex.ts

```diff
@@ -331,14 +333,16 @@ export default class SmoothRepresentationIndex
         time: 0,
         timescale: index.timescale,
         privateInfos: {
-          type: "smooth-init",
-          bitsPerSample: this._bitsPerSample,
-          channels: this._channels,
-          codecPrivateData: this._codecPrivateData,
-          packetSize: this._packetSize,
-          samplingRate: this._samplingRate,
-          protection: this._protection,
+            smoothInit: {
+              bitsPerSample: this._bitsPerSample,
+              channels: this._channels,
+              codecPrivateData: this._codecPrivateData,
+              packetSize: this._packetSize,
+              samplingRate: this._samplingRate,
+              protection: this._protection,
+            },
         },
+        media: "",
```

Comment:
why though?

I'm not OK with that. Here media is not defined or is null. It is not an empty string